### PR TITLE
Updating our chains data

### PR DESCRIPTION
**Pull request by stepanLav**
# This PR is updating our data in chains.

1. Was found that Patract's nodes were deprecated for all networks, they were replaced by Dwellir's nodes which were added in polkadot.js/apps
2. As some nodes were out of date, the [script](https://github.com/nova-wallet/nova-utils/pull/336/files#diff-05f5624502aae9b0670e5eba0741583fb2dcbc2f3496fa09ca753b6c3c8bf402) was created to check connecting possibility for each node, which was added to our config. 
3. As some data was different in dev and prod configs, the [script](https://github.com/nova-wallet/nova-utils/pull/336/files#diff-a84d22f78088e4f0adc746a94542a4b1ae8a990a9947d738f241fde475d2d089) was created to compare these files.
4. As a result of running the script from point 3 of this list, some changes have been made.

P.S. all scripts were added in the tests folder, as it would be great to automate it in future

### CODE REVIEW

**Review by leohar — APPROVED** [review]

**Comment by leohar** [comment]
maybe it makes sense in future to separate tests from json update in different pr

File: tests/test_network_availability.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from distutils.log import error
+import json
+from substrateinterface import SubstrateInterface
+
+def create_connection(url):
```
Comment (valentunn) [inline tests/test_network_availability.py:5]: We can setup a CI check with this actually

File: chains/v3/chains_dev.json [hunk 2]
```diff
@@ -1867,7 +1871,12 @@
                 "symbol": "GLMR",
                 "precision": 18,
                 "priceId": "moonbeam",
-                "staking": "parachain"
+                "staking": "parachain",
+                "buyProviders": {
+                    "transak": {
+                       "network": "MOONRIVER"
```
Comment (antonkhvorov) [inline chains/v3/chains_dev.json:1877]: Most likely it should be "MOONBEAM" or "MAINNET", I'm not sure

Comment (antonkhvorov) [reply chains/v3/chains_dev.json:1877]: in addition to that, make sure to fix that in chains.json as well

Comment (antonkhvorov) [reply chains/v3/chains_dev.json:1877]: I bet it should be "MAINNET" according to:
https://www.notion.so/d20db13dcacb4ec8801e8869e2e5975e?v=9bcdb10756834041811b64d527b5fdc1

File: tests/test_network_availability.py (hunk 1 above)
Comment (stepanLav) [reply tests/test_network_availability.py:5]: Yes, that's right, I thought do that later with a little bit more complex checks.

**Review by antonkhvorov — CHANGES_REQUESTED** [review]

File: chains/v3/chains.json [hunk 3]
```diff
@@ -1363,20 +1371,16 @@
         ],
         "nodes": [
             {
-                "url": "wss://falafel.calamari.systems/",
-                "name": "Manta Network node 0"
+                "url": "wss://ws.calamari.systems/",
+                "name": "Manta Network"
```
[the remark this answers is not in the document]
Comment (stepanLav) [reply chains/v3/chains.json]: Fixed

**Review by antonkhvorov — APPROVED** [review]

**Pull request merged**